packages/config/config/devices/0x0212/gda-a2e6b-k0.json

```diff
@@ -0,0 +1,88 @@
+// GATEMAN GDA-A2E6B-K0
+// A20-SH
+{
+	"manufacturer": "GATEMAN",
+	"manufacturerId": "0x0212",
+	"label": "GDA-A2E6B-K0",
+	"description": "A20-SH",
```

Comment:
Should that be part of the label? If not, either remove it or find a real description :)